docs/quickstart.md

```diff
@@ -61,6 +61,26 @@ If you don't like `npm` or have trouble installing the tool, you can manually cr
 </html>
 ```
 
+### Specifying docsify versions
```

Comment:
Insert above this header, under the HTML block:

Specifying a major version in the URL (@4) will allow your site will receive non-breaking enhancements (i.e. "minor" updates) and bug fixes (i.e. "patch" updates) automatically. This is the recommended way to load docsify resources.